scripts/GenerateJSONOutput.py

```diff
@@ -50,25 +57,12 @@
 
 args = parser.parse_args()
 
-
 # Print out the settings
-for setting in dir(args):
-    if not setting[0] == "_":
```

Comment:
Note sure why this check was ever needed, but happy to have it back in there if really needed